Makefile

```diff
@@ -9,7 +6,8 @@ help: ## Print the help message
 
 .PHONY: run
 run: ## Run the app
-	FLASK_APP=$(FLASK_APP) flask run --debug -h localhost -p 5000
+	@source ./env.sh ; \
```

Comment:
The semicolon should be `&&` so that if the `source` fails the target fails.

Reply:
✅